testData/dbData/views.json

```diff
@@ -11,6 +11,18 @@
             "findAuthorizationByAccessToken": {
                 "map": "function(doc) {if (doc.type === 'gpiiAppInstallationAuthorization' && doc.revoked === false) {emit(doc.accessToken, {'_id': doc.clientId, 'authorization': doc})}}"
             },
+            "findKeysForPrefsSafe": {
+                "map": "function(doc) {if (doc.type === 'gpiiKey' || doc.type === 'gpiiCloudSafeCredential') emit(doc.prefsSafeId, null)}"
```

Comment:
This view returns not only GPII keys but also safe credentials. Is this intentional?

Reply:
It is, and ideally I want it to return any future related docs and credential types we come up with, such as Nova tokens or anything else.  Renamed to findRelatedDocsForPrefsSafe